compiler/env/OMRCPU.hpp

```diff
@@ -107,13 +117,19 @@ class OMR_EXTENSIBLE CPU
     */
    static TR::CPU detect(OMRPortLibrary * const omrPortLib);
 
-  /** 
+   /** 
     * @brief Returns the processor type and features that will be used by portable AOT compilations
     * @param[in] omrPortLib : the port library
     * @return TR::CPU
     */
    static TR::CPU detectRelocatable(OMRPortLibrary * const omrPortLib);
 
+   /**
+    * @brief Intialize _featureMasks to the list of processor features that will be utilized by the compiler and set _isFeatureMasksEnabled to true
+    * @return void
+    */
```

Comment:
Setting of `_isFeatureMasksEnabled` seems like an implementation detail. I don't think we should document this on an external API.